pkgs/servers/home-assistant/.gitignore

```diff
@@ -0,0 +1 @@
+.mypy_cache/
```

Comment:
I would put this in `.git/info/exclude`
```suggestion
```

Reply:
We have some mypy stuff now in nixpkgs. Maybe we can also add it to the global .gitignore.